<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_image_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>250</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_image_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>250</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_start">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stop">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_init">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_time_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>time_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_time_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>time_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_1">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>name_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_2">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>name_2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
